modules/migration-state-migration-cli.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/about-mtc-3-4.adoc
+// * migration_toolkit_for_containers/about-mtc.adoc
+
+[id="migration-state-migration-cli_{context}"]
+= Migrating an application's state
+
+You can use {mtc-full} ({mtc-short}) to migrate an application's state. {mtc-short} copies persistent volume claims (PVCs) and specific Kubernetes objects, for example, `Secret` custom resources (CRs) or config maps that store an application's state.
+
+If you have a CI/CD pipeline, you can migrate stateless components by deploying them on the target cluster. Then you can migrate the state of the application by selecting PVCs and Kubernetes objects.
+
+You can migrate PV data from the source cluster to PVCs that are already provisioned in the target cluster by mapping PVCs in the `MigPlan` CR. This ensures that the target PVCs of migrated applications are synchronized with the source PVCs.
```

Comment:
I am confused this description, the PVCs don't need to be provisioned in the target cluster  before Stage.

> You can migrate PV data from the source cluster to PVCs that are already provisioned in the target cluster by mapping PVCs in the `MigPlan` CR

Reply:
Pranav wrote that in the upstream docs. If this is not the case, I will remove it.